aws/auth/README.md

```diff
@@ -17,13 +17,18 @@ AWS role ARN we use for Elastic Observability repositories.
 | `aws-account-id`        | The AWS account ID                                                                                                               | `false`  | `697149045717` |
 | `aws-region`            | The AWS region, e.g. us-east-1                                                                                                   | `false`  | `us-east-1`    |
 | `role-duration-seconds` | The assumed role duration in seconds, if assuming a role. Defaults to 1 hour, but cannot exceed the maximum defined by the role. | `false`  | `3600`         |
+| `output-credentials`    | Whether to output the AWS credentials                                                                                            | `false`  | `false`        |
```

Comment:
maybe always true and we don't need this?

Reply:
I was also thinking about this and wondered what the reason for setting it to `false` by default in the upstream action is.

But since you mentioned it, I will make it true by default for us.

Reply:
https://github.com/elastic/oblt-actions/pull/171/commits/ab3cf6e53228558bc79da0fb274d6155fb4e3447